released/discovery_center/mission_4327/setup_subaccount_cf/samples.tfvars

```diff
@@ -0,0 +1,29 @@
+# ------------------------------------------------------------------------------------------------------
+# Provider configuration
+# ------------------------------------------------------------------------------------------------------
+# Your global account subdomain
+globalaccount        = "youraccount"
+region               = "ap11"
+subaccount_name      = "subaccount name"
+btp_user = "jane.doe@test.com"
+# ------------------------------------------------------------------------------------------------------
+# Project specific configuration (please adapt!)
+# ------------------------------------------------------------------------------------------------------
+subaccount_admins         = ["jane.doe@test.com", "john.doe@test.com"]
```

Comment:
if you just replicate the values of the defaults you can either omit the defaults or leave them out from the samples.tfvars